Client/Frontend/Brave Notifications/Brave Wallet/WalletNotification.swift

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+public enum WalletNotificationAction {
```

Comment:
Same comment above `public` as above
Same comment about nesting it within the notification and calling it `Action`